backend/src/controllers/index.js

```diff
@@ -1,57 +1,28 @@
+import {fetchData, findInObject} from "../helper";
+const opportunities = require('../helper/data.json');
```

Comment:
@fcoskunn  can we use `import` instead of required so we are consistent 
```suggestion
import opportunities from '../helper/data.json';
```

Reply:
It escaped me, I have just changed.